src/Windows/Core/Impl/Microsoft.Rtvs.Core.Windows.csproj

```diff
@@ -67,7 +67,7 @@
     <Compile Include="Shell\CoreShell.cs" />
     <Compile Include="Shell\CoreShellExtensions.cs" />
     <Compile Include="Shell\ICompositionCatalog.cs" />
-    <Compile Include="Shell\ICoreShell.cs" />
```

Comment:
When you split the editor, we need to check if we really need `ICoreShell` in cross-plat